content/english/python-interactive/activity-5.md

```diff
@@ -0,0 +1,36 @@
+  - Still within the for loop, we now need to add this to the dictionary `dictItems`. However, remember that since a customer can technically order the same thing multiple times in separate order items, we need to combine that here. For instance, if the customer orders Spring Rolls-2 twice, we want to interpret that as 4 Spring Rolls. To do this, we need to have an if-else block where if the dictionary already contains the item name, then add this new quantity to the quantity already associated with the item name, else create a new key-value pair with the item name and quantity.
+  - Finally, after this loop ends, remember to return this dictionary!
+
+### Compute prices from the dictionary  
+The second function we're going to write is `computePrices(dictItems)` which takes the dictionary we created in the last function as a parameter and returns a new dictionary that has every item name associated with the price * quantity.
+  - Create an empty dictionary `dictItemAndPrice` will hold the item and the total price for that item.
+  - Loop through the items in the dictionary we just passed in. Within this loop, we want to get the item quantity from `dictItems` since it's the value in the dictionary's key-value pair. We also want to access the item's price from the global variable `dictMenu`. Keep in mind though that it is the first value in the list associated with the item name and we need to access it accordingly.
+  - Multiply those two numbers together to get the total price for that item.
+  - Add the item name and total price pair to the dictionary `dictItemAndPrice`.
+  - Return this dictionary.
+
+### Compute the total price of the order
+The third function we'll create is `computeTotalPrice(dictItemAndPrice)` which takes the dictionary we created in `computePrices(dictItems)` as a parameter and returns the total price of the customer's entire order.
+  - Initialize a variable `totalPrice` to 0 so we can add the prices of each item to it.
+  - Loop through the dictionary `dictItemAndPrice` and add each of the values to `totalPrice`.
+  - Return totalPrice rounded to 2 decimal places by using `round()` and provide 2 as the second value so it rounds to the hundredths place since that's how we want the final price to be displayed.
+
+All 3 of these helper functions will be called in the `main()` in the else block in the order we wrote them in. The value we returned for the 1st function will be the argument to the second function, and the value returned from the 2nd function will be the argument to the third function, and finally, the result of the 3rd function will be used in the next activity! Remember to explicitly store each of the results of the functions in variables so that you can reuse them multiple times.
```

Comment:
```suggestion
All 3 of these helper functions will be called in the `main()` function in the `else` block in the order we wrote them in. The value we returned for the 1st function will be the argument to the second function, and the value returned from the 2nd function will be the argument to the third function, and finally, the result of the 3rd function will be used in the next activity! Remember to explicitly store each of the results of the functions in variables so that you can reuse them multiple times.
```